tools/fileinfo/features/pe-delayed-imports/test.py

```diff
@@ -47,7 +46,7 @@ def test_delayed_imports_detection(self):
         self.assertEqual(self.fileinfo.output['importTable']['imports'][77]['address'], '0x4020f2')
         self.assertEqual(self.fileinfo.output['importTable']['imports'][78]['address'], '0x4020c0')
         self.assertEqual(self.fileinfo.output['importTable']['imports'][79]['address'], '0x4020e0')
-        self.assertEqual(self.fileinfo.output['importTable']['imports'][69]['ordinalNumber'], '20')
+        self.assertEqual(self.fileinfo.output['importTable']['imports'][69]['name'], 'sendto')
```

Comment:
Do I understand it correctly this is related to RETDEC-61? It says something about names and ordinals being mutually exclusive, but I'm not sure about all the implications. Did we just detect `sendto` now instead of `20`? Is that value `20` still somewhere? Shouldn't we test for it?
RETDEC-61 talks about some hints. Do we have some entry for this? If no, shouldn't we have it? If yes, couldn't we test for it?

Reply:
This is related to RETDEC-74. Because YARA artificially replaces some of the ordinals by function names, we need to do the same. After implementing the change, imports from WS2_32.dll (Windows implementation of Winsock) now contain name instead of ordinals.

Reply:
When an import name is artificially supplied (YARA-like behavior), both JSON and plain representation contains both import name AND ordinal. This test will be fixed so both are verified.